# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../kdebindings/phonon_includes.patch
# Copyright (C) 2010 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur kdebindings-4.4.0-orig/smoke/phonon/phonon_includes.h kdebindings-4.4.0/smoke/phonon/phonon_includes.h
--- kdebindings-4.4.0-orig/smoke/phonon/phonon_includes.h	2010-02-20 17:00:18.471892638 +0000
+++ kdebindings-4.4.0/smoke/phonon/phonon_includes.h	2010-02-20 17:02:41.202908366 +0000
@@ -4,8 +4,8 @@
 #include <phonon/abstractmediastream.h>
 #include <phonon/abstractvideooutput.h>
 #include <phonon/addoninterface.h>
-#include <phonon/audiodataoutput.h>
-#include <phonon/audiodataoutputinterface.h>
+#include <phonon/experimental/audiodataoutput.h>
+#include <phonon/experimental/audiodataoutputinterface.h>
 #include <phonon/audiooutput.h>
 #include <phonon/audiooutputinterface.h>
 #include <phonon/backendcapabilities.h>
